lib/nexthop.h

```diff
@@ -58,6 +58,62 @@ enum blackhole_type {
 
 enum nh_encap_type {
 	NET_VXLAN = 100, /* value copied from FPM_NH_ENCAP_VXLAN. */
+	FPM_NH_ENCAP_SRV6_ROUTE = 101,
+	FPM_NH_ENCAP_SRV6_LOCAL_SID = 102,
+};
+
+enum {
```

Comment:
not crazy about these anonymous enums - do we have guidance about that in the dev docs?